PhysicsTools/JetMCAlgos/python/AK4GenJetFlavourInfos_cfi.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.JetMCAlgos.AK4PFJetsMCFlavourInfos_cfi import ak4JetFlavourInfos
+#from PhysicsTools.PatAlgos.slimmedGenJets_cfi import slimmedGenJets
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant